### ISSUE TITLE
Capture geometry (waypoint) to support the creation of an observation/treatment

### ISSUE THREAD

**Issue by ghost** [bcgov/invasivesbc#122]
As a general user I want to be able to capture a geometry type (specifically a waypoint path) and have that geometry reflected as in the observation/treatment record.

**Additional Context**
•	This collection methods to be made available in the mobile field collection application
- [x]	An initial anchor point for the observation is required
- [x]	The user is required to enter a sequence of points from the anchor along a linear path. (This can be specified using the stylus as POC. Using the device GPS is secondary requirement)
- [x] 	The minimum number of points is 2 (the anchor plus 1) 
- [ ]          Max 20 total points.
- [ ] 	As each point is captured, the distance to the previous point should be greater than 1 meter for that point to be retained
- [x] 	The user is prompted to enter an offset distance that provides width to the path (Minimum of 1 meter to a maximum of 10 meters.) "the user can provide 1 metre (default) to up to 10m total (so 0.5 m to 5m for either side"
- [x] 	A polygon is generated about that waypoint path using that offset to both sides of the path and an area calculated
- [ ] 	the resulting anchor point and polygon appear on the map. The user can accept or reject the geometry, if there is the ability to edit the buffer and the actual line this is desirable. 

**Acceptance Criteria**
- [x] Given the anchor point, When confirmed, Then the user is prompted to record the waypoints via on screen capture or stylus
- [x] Once the path is complete, the user is prompted for an offset
- [x] With the offset, calculate the resultant polygon and calculate the area

nice to have for this ticket or can be done in another:
- [ ] The anchor point and polygon are draw accordingly. The user can accept or re-type in a new offset value.

**Comment by micheal-w-wells** [comment]
@mikeshasko Given the line turns into a polygon, the number of (and length between) points is mute.  Given we have one geometry per record, you aren't going to see the anchor point displayed along with the polygon.  Can you confirm that the boxes checked cover off what we can call v1 of waypoint?
